crypto/merklesignature/merkleSignatureScheme_test.go

```diff
@@ -240,7 +241,7 @@ func TestSigning(t *testing.T) {
 
 	err = signer.GetVerifier().Verify(start+5, hashable, sig)
 	a.Error(err)
-	a.Contains(err.Error(), ErrSignatureSchemeVerificationFailed)
+	a.True(errors.Is(err, ErrSignatureSchemeVerificationFailed))
```

Comment:
nit: newer versions of testify have ErrorIs https://pkg.go.dev/github.com/stretchr/testify/require?utm_source=godoc#ErrorIs